### ISSUE TITLE
Unexpected behaviour on autocomplete dropdowns and mouse hover

### ISSUE THREAD

**Issue by SirVeggie** [microsoft/vscode#129016]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.57.1
- OS Version: Windows_NT x64 10.0.19042

### Expected behaviour:

Autocomplete dropdown ignores mouse until it is moved (which indicates intent on selecting with a keyboard vs mouse).

### Actual behaviour:

When opening an autocomplete dropdown, the dropdown selection is instantly focused on the mouse location instead of the first item.

### Steps to Reproduce:

1. Move mouse to the area where an autocomplete dropdown will appear
2. Trigger autocomplete
3. A 'random' item (the one hovered over by the mouse) from dropdown is focused instead of the first item

### Why this is bad:

When I trigger an autocomplete without even touching my mouse, I expect the focus to be on the first item of the list. When I am like 90% sure I want to select the first item I might even pre-emptively press tab to select it without properly looking at the focus position as I expect it to be on the first item.

This causes accidental selects of unintended items of the dropdown list, which is quite annoying.

Note: If mouse is not involved, the selection always starts at the first item due to my vscode settings.

**Comment by jrieken** [comment]
> When I trigger an autocomplete without even touching my mouse, I expect the focus to be on the first item of the list. When I am like 90% sure I want to select the first item I might even pre-emptively press tab to select it without properly looking at the focus position as I expect it to be on the first item.

The mouse only focuses items but it doesn't select them. Selected items will be inserted, not focus items. What you are saying isn't reproducible - the selected item is always inserted, not the focused one


https://user-images.githubusercontent.com/1794099/126280983-d28111e8-ed3f-482c-92c1-06c84454ee30.mov

**Comment by SirVeggie** [comment]
You're right, I tried it again, and it does not occur. I have a distinct memory of opening autocomplete, and having to use arrow keys to move the selection back to the first one from where the mouse is located multiple times in the recent past. However now it doesn't seem to happen so I'm feeling very puzzled. I suppose I'll close this issue since I can't even reproduce it myself.

Perhaps I'm just slowly descending into madness :)

**Comment by SirVeggie** [comment]
Sorry, I just happened upon this issue again (after coming back to a project) and now I realized what I got wrong in my initial issue.

The issue I described wasn't about the autocomplete dropdown, but rather the code actions dropdown which is triggered by the hotkey [Ctrl + .]

I'll modify the initial issue and reopen it.

![image](https://user-images.githubusercontent.com/32365239/128639219-525fb753-ac44-4280-885f-b0ebdb831f91.png)
